packages/website/src/features/Packages/interact/ContractsList.tsx

```diff
@@ -91,42 +117,65 @@ export const ContractsList: FC<ContractsListProps> = ({
                   <MoreHorizontal className="h-4 w-4 text-white" />
                 </div>
               </PopoverTrigger>
-              <PopoverContent className="max-h-[320px] max-w-[320px] overflow-y-auto overflow-x-hidden w-full bg-background border border-border p-0">
-                {otherOptions.length > 5 && (
-                  <div className="p-2">
-                    <SearchInput size="sm" onSearchChange={setSearchTerm} />
+              <PopoverContent className="max-h-[320px] max-w-[350px] overflow-y-auto overflow-x-hidden w-full bg-background border border-border p-0">
+                {otherOptions.map((option, i) => (
+                  <div
+                    key={i}
+                    className={`cursor-pointer p-2 border-t border-border font-mono ${
+                      isActiveContract(option)
+                        ? 'bg-background'
+                        : 'bg-transparent'
+                    } hover:bg-accent/50`}
+                    onClick={async () => {
+                      setIsPopoverOpen(false);
+                      await router.push(
+                        buildInteractPath(
+                          name,
+                          tag,
+                          variant,
+                          option.moduleName,
+                          option.contractName,
+                          option.contractAddress
+                        )
+                      );
+                    }}
+                    data-testid={`${option.contractName}-button`}
+                  >
+                    <span className="text-sm">
+                      {`${option.moduleName}.${option.contractName}`}
```

Comment:
so yea here lets display `option.contractName` first, and hten in separate, smaller, gray text, the module name